src/components/apiComponents/endPointSelector.jsx

```diff
@@ -0,0 +1,94 @@
+import { Select } from '@headlessui/react'
+import { ChevronDownIcon } from '@heroicons/react/20/solid'
+import clsx from 'clsx'
+import { act, useState } from 'react'
+
+const endPoints = {
+  eclipseAuraMainnet: {
+    name: 'Eclipse Mainnet',
+    uri: 'https://aura-eclipse-mainnet.metaplex.com',
+    value: 'eclipseAuraMainnet',
+  },
+  custom: {
+    name: 'Custom',
+    uri: 'custom',
+    value: 'custom',
+  },
```

Comment:
Is it a problem for us to show 'https://aura-mainnet.metaplex.com' right now when they select "Custom"?  For example we only show the beta signup on the Solana RPC page.

Reply:
I don't think it's ready for beta/testing? The end points were removed from the main docs so I assumed they wouldn't go live with this either.